modules/serverless-config-replicas-eventing.adoc

```diff
@@ -0,0 +1,49 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-HA.adoc
```

Comment:
This isn't your fault but reminds me, can you do me a huge favor and change that assembly name from `serverless-HA.adoc` to `serverless-ha.adoc`? I've been meaning to but just busy with other things. You should be able to do a git grep to find any instances 🙂

Reply:
The huge, gigantic favor is done :vcorleone: